app/src/main/res/values/strings_i18n.xml

```diff
@@ -388,7 +388,7 @@ from friends yet.</string>
   <string name="Opens_email_composer" formatted="false">Opens email composer.</string>
   <string name="Opens_filters" formatted="false">Opens filters.</string>
   <string name="Opens_help_sheet" formatted="false">Opens help sheet.</string>
-  <string name="open_inbox" formatted="false">Open Inbox</string>
```

Comment:
I've just realised about this :sweat_smile:, but, is the string "Open inbox" translated? Or we haven't done the translations yet? @leighdouglas